src/mon/MonCommands.h

```diff
@@ -322,6 +322,9 @@ COMMAND_WITH_FLAG("mds set " \
 	"name=val,type=CephString "					\
 	"name=yes_i_really_mean_it,type=CephBool,req=false",			\
 	"set mds parameter <var> to <val>", "mds", "rw", FLAG(OBSOLETE))
+COMMAND_WITH_FLAG("mds frozen name=role,type=CephString"
```

Comment:
nit: it seems like this should be 'mds freeze ...' not 'mds frozen ...' since it's taking an action "freeze!", not reporting state ("is it frozen?")

Reply:
Ya I was thinking `freeze .. false` sounded silly but the reverse is also true. I'll use freeze.

Reply:
fixed